src/test/feature/run-feature-test.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+if [ x$GPHOME == 'x' ]; then
+  echo "Please export GPHOME variable."
+  exit 0
+fi
```

Comment:
I think we need more words. How about this?

Please export the environment variable GPHOME before running.
Typically you can run " source greenplum_path.sh" under
the hawq installation path.